lib/src/runtime/composables/useAsyncStoryblok.ts

```diff
@@ -0,0 +1,36 @@
+import { useStoryblokApi, useStoryblokBridge } from "@storyblok/vue";
+import type { ISbStoriesParams, StoryblokBridgeConfigV2, ISbStoryData, ISbError, ISbResult } from '@storyblok/vue';
+import { useAsyncData, useState, onMounted, createError } from "#imports";
+
+export const useAsyncStoryblok = async (
+  url: string,
+  apiOptions: ISbStoriesParams = {},
```

Comment:
Hey @Dawntraoz, `apiOptions` and `bridgeOptions` have a default value because we need to pass them as empty below on the `storyblokApiInstance.get(`cdn/stories/${url}`, apiOptions)` and `useStoryblokBridge` respectively?

Reply:
Well the bridge is already initializing the options to empty object here -> https://github.com/storyblok/storyblok-js/blob/main/lib/index.ts